package/yast2-perl-bindings.spec

```diff
@@ -74,8 +74,6 @@ rm $RPM_BUILD_ROOT/%{perl_vendorarch}/auto/YaST/YCP/libYCP.la
 %{perl_vendorarch}/auto/YaST/YCP/libYCP.so*
 %dir %{yast_moduledir}
 %{yast_moduledir}/YaPI.pm
-%{yast_ydatadir}/devtools/pluglib-bindings
```

Comment:
pluglib-bindings are NOT the legacy testsuite

They can be probably dropped too, but they're a very different thing. They were used for LiMaL which was dropped after SLE 12.

Reply:
OK, that was already proposed in #22. Meanwhile I have re-checked that nothing uses pluglib-bindings. Just mention it in the changelog and the PR description.

Reply:
> Just mention it in the changelog and the PR description.

I'll do, thanks!